lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -407,8 +408,8 @@ export abstract class CacheManager implements ICacheManager {
         environment: string
     ): boolean {
         if (
-            EnvironmentAliases.includes(environment) &&
-            EnvironmentAliases.includes(entity.environment)
+            TrustedAuthority.getCloudDiscoveryMetadata(environment) &&
+            TrustedAuthority.getCloudDiscoveryMetadata(entity.environment)
```

Comment:
check for same alias list, not the entire lists.

Reply:
Done